src/sidebar/test/annotation-ui-test.js

```diff
@@ -507,6 +507,15 @@ describe('annotationUI', function () {
     });
   });
 
+  describe('#updateFlagStatus', function () {
```

Comment:
This test is fine but we should migrate tests for annotation actions into the relevant reducer module (`reducers/test/annotations_test`).